agimus_controller_ros/package.xml

```diff
@@ -1,21 +1,40 @@
 <?xml version="1.0"?>
-<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
 <package format="3">
   <name>agimus_controller_ros</name>
   <version>0.0.0</version>
-  <description>TODO: Package description</description>
-  <maintainer email="kateryna.zorina@cvut.cz">gepetto</maintainer>
-  <license>Apache-2.0</license>
+  <description>The agimus_controller package</description>
 
-    <!-- Build dependencies -->
+  <maintainer email="guilhem.saurel@laas.fr">Guilhem Saurel</maintainer>
+  <author email="tmartinezf@laas.fr">Théo Martinez</author>
+
+  <license>BSD-3-clause</license>
+  <url type="website">https://github.com/agimus-project/agimus_controller</url>
+
+
+  <build_depend>jrl-cmakemodules</build_depend>
+  <!-- The following tags are recommended by REP-136 -->
+  <exec_depend condition="$ROS_VERSION == 1">catkin</exec_depend>
+  <exec_depend condition="$ROS_VERSION == 2">ament_cmake</exec_depend>
+
+  <!-- ROS dependencies. -->
+  <depend>linear_feedback_controller_msgs</depend> <!-- install manually. -->
```

Comment:
Install manually should be specified as a VCS-style dependency

Reply:
So what should be written exactly in this line ?